### PR TITLE
Imperia fallonar to dos pre 1.7

### PR DESCRIPTION
**Pull request by Unintuition-Time**
Includes law rebalancing, unit/mop rebalancing, subsistence fix, and pan-nationalism requirement for major formables

### CODE REVIEW

File: common/country_formation/00_major_formables.txt
Comment (Danarca) [inline common/country_formation/00_major_formables.txt:1]: Remember to do a commit with just the uploaded file, unchanged.
That allows us to see what you've *actually* changed in the file.

File: common/country_formation/00_major_formables.txt
Comment (Danarca) [inline common/country_formation/00_major_formables.txt]: So.. Now any italian country just needs pan-nationalism, instead of all italians having nationalism?
I agree with the pan-nationalism change, but.. It feels off.

File: common/production_methods/11_private_infrastructure.txt
Comment (Danarca) [inline common/production_methods/11_private_infrastructure.txt:10]: I'm sorry, what?

File: common/production_methods/13_construction.txt
Comment (Danarca) [inline common/production_methods/13_construction.txt]: The numbers, Fallonar! What do they mean!?

File: common/mobilization_options/Imperia_mobilization_option.txt
Comment (Danarca) [inline common/mobilization_options/Imperia_mobilization_option.txt:1]: Quite a few indentation errors in this file, if you can't get the autoformatter to set it up, perhaps I can be of use? :)

**Review by Danarca — CHANGES_REQUESTED** [review]

File: common/country_formation/00_major_formables.txt
Comment (Unintuition-Time) [reply common/country_formation/00_major_formables.txt:1]: Yea I apologize about that, was one of the first things I did so I was rusty. It was only changing Germany + Italy formations to pan-nationalism instead of nationalism

File: common/production_methods/11_private_infrastructure.txt
Comment (Unintuition-Time) [reply common/production_methods/11_private_infrastructure.txt:10]: profitability calculation I made in a note regarding railways. mostly for myself for later.

File: common/production_methods/13_construction.txt
Comment (Unintuition-Time) [reply common/production_methods/13_construction.txt]: Sorry, I'll clarify the numbers' meaning XD

File: common/mobilization_options/Imperia_mobilization_option.txt
Comment (Unintuition-Time) [reply common/mobilization_options/Imperia_mobilization_option.txt:1]: I'll take a look at formatting it later today

File: common/country_formation/00_major_formables.txt
Comment (Unintuition-Time) [reply common/country_formation/00_major_formables.txt]: You think we should make it a percentage too? I can work with that, was only concerned that the ai might lag on tech and prevent nation formation as a result

File: common/production_methods/13_construction.txt
Comment (HiddeLekanne) [inline common/production_methods/13_construction.txt:8]: further construction nerves for low tech countries? Are we really experiencing disproportionate growth from low tech countries to necessitate this?

File: common/combat_unit_types/00_combat_unit_types.txt
Comment (HiddeLekanne) [inline common/combat_unit_types/00_combat_unit_types.txt]: not your current code, but would be nice to format when it is within view.

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: common/production_methods/13_construction.txt
Comment (Unintuition-Time) [reply common/production_methods/13_construction.txt:8]: Further? we reverted them for this patch. I am reverting the revert to bring it back to what it was in our previous iterations because the reasons we reverted (fears of higher construction costs for buildings like railways) ended up not being enough to stop the construction tide.

File: common/technology/technologies/eras/00_eras.txt
Comment (Danarca) [inline common/technology/technologies/eras/00_eras.txt:1]: Any specific reason we're making these "faster research speed, higher research costs" changes?
I've not been following the past conversations surrounding research too closely, so I'll probably trust your judgement on this, but I'm still curious :)

File: common/country_formation/00_major_formables.txt
Comment (Danarca) [inline common/country_formation/00_major_formables.txt:55]: Perhaps also make a check on the unifier having the tech? I know, they are very unlikely to *not* have it at this point, but still..

File: common/technology/technologies/eras/00_eras.txt
Comment (Unintuition-Time) [reply common/technology/technologies/eras/00_eras.txt:1]: We removed the research speed penalty because the goal is to make techs more expensive, but research speed only affects innovation and not tech spread, so we simply migrated over the goal into eras.

Comment (Danarca) [reply common/technology/technologies/eras/00_eras.txt:1]: Ahh, that makes sense!

**Review by Danarca — APPROVED** [review]
Not sure if you need both of us to approve before you can merge, but do wait for Kikkers approval, just in case :)

File: common/technology/technologies/eras/00_eras.txt
Comment (HiddeLekanne) [reply common/technology/technologies/eras/00_eras.txt:1]: we could also add it to tech spread, but at that point why not get rid of a fixed static_modifier that obscures the tech costs from the player.

File: common/production_methods/13_construction.txt
Comment (HiddeLekanne) [reply common/production_methods/13_construction.txt:8]: ah yeay nevermind that. I just wanna stress that we need to account for the average (quite nooby) player, not the min-maxer. But we will see whether this is too much or too little.

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**
